tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -155,6 +156,20 @@ def assertSuccessfulRpcs(self,
             0,
             msg=f'Expected all RPCs to succeed: {failed} of {num_rpcs} failed')
 
+    def assertXdsConfigExists(self, test_client: XdsTestClient):
+        config = test_client.csds.fetch_client_status()
+        self.assertIsNotNone(config)
+        seen = set()
+        want = frozenset([
+            'listener_config', 'cluster_config', 'route_config',
+            'endpoint_config'
+        ])
+        for xds_config in config.xds_config:
+            seen.add(xds_config.WhichOneof('per_xds_config'))
+        logger.debug('Received xDS config dump: %s',
+                     json_format.MessageToJson(config, indent=2))
+        self.assertEqual(want, seen)
```

Comment:
```suggestion
        self.assertSameElements(want, seen)
```

This one has a bit nicer output. F.e `assertSameElements()` output if `endpoint_config` missing, and `unexpected_config` present:

```
AssertionError: Expected, but missing:
  ['endpoint_config']
Unexpected, but present:
  ['unexpected_config']
```

Vs `assertEqual()`:
```
AssertionError: {'endpoint_config', 'listener_config', 'c[26 chars]fig'} != frozenset({'listener_config', 'unexpected[39 chars]ig'})
```

Reply:
TIL. Good idea!